From b3dd9ae416e8dfa516faadf91e7d0aefe57aa376 Mon Sep 17 00:00:00 2001
From: Roman Shaposhnik <rvs@zededa.com>
Date: Tue, 25 Aug 2020 08:49:21 +0530
Subject: [PATCH] Monster of a workaround for Xen on Dell 300x boot issues

This change is pulled in from the eve.git commit:
794721419fa1886d9b628803b5fa8 ("Monster of a workaround for Xen on Dell 300x boot issues")
---
 drivers/pinctrl/intel/pinctrl-baytrail.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/pinctrl/intel/pinctrl-baytrail.c b/drivers/pinctrl/intel/pinctrl-baytrail.c
index f38d596e..516e8b0b 100644
--- a/drivers/pinctrl/intel/pinctrl-baytrail.c
+++ b/drivers/pinctrl/intel/pinctrl-baytrail.c
@@ -1723,6 +1723,7 @@ static int byt_gpio_probe(struct byt_gpio *vg)
 		return ret;
 	}
 
+#if 0
 	/* set up interrupts  */
 	irq_rc = platform_get_resource(vg->pdev, IORESOURCE_IRQ, 0);
 	if (irq_rc && irq_rc->start) {
@@ -1738,6 +1739,7 @@ static int byt_gpio_probe(struct byt_gpio *vg)
 					     (unsigned)irq_rc->start,
 					     byt_gpio_irq_handler);
 	}
+#endif
 
 	return ret;
 }
-- 
2.17.1

